manifests/profile/networking/firewall/http.pp

```diff
@@ -1,13 +1,20 @@
-# Copyright (c) 2018 The Regents of the University of Michigan.
+# Copyright (c) 2020 The Regents of the University of Michigan.
```

Comment:
Having correct but redundant copyright dates on files that already have a rich git history is very hard to care about, so I'm not requesting a change here, but in general I've seen these dates used to show when a file has changed. So `copyright 2015` means the file hasn't changed since 2015, `copyright 2015, 2018` means it was created in 2015 and later modified in 2018, while `copyright 2015-2018` means the file was undergoing constant change in 2015, 2016, 2017, and 2018.

So in this case, rather than change `2018` to `2020` (implying the file was created in 2020), the syntax would be to change from `2018` to `2018, 2020` (created in 2018, left alone in 2019, and modified in 2020).

But I've yet to see us have a clear policy around that, nebula is full of incorrect uses, and, as I said, there's also a git history, so my intent with this comment is purely to be informative.

Reply:
Is there a way to automate it? Cause that would help consistency and reduce tedium.